api/FontFace.json

```diff
@@ -128,7 +128,7 @@
               "version_added": "11"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "10"
```

Comment:
Is it really right that this was added in Safari iOS 10, but in Safari 11 for macOS? That the above "11" wasn't updated to "10" suggests that, or was the desktop data not used for this PR?

Reply:
I did some more testing, and it looks like there's some odd issue with the FontFace tests in Safari 11, so the collector didn't adjust the version for this.  I did some manual testing and confirmed it was supported since Safari 10.

Reply:
Excellent, thank you!